modules/phase_field/src/userobjects/DiscreteNucleationFromFile.C

```diff
@@ -91,9 +110,18 @@ DiscreteNucleationFromFile::DiscreteNucleationFromFile(const InputParameters & p
     paramError("file", "Missing 'y' column in file");
   if (!found_z && _mesh.dimension() >= 3)
     paramError("file", "Missing 'z' column in file");
+  if (!found_r && !_fixed_radius)
+    paramError("file", "Missing 'r' column in file");
+  if (found_r && _fixed_radius)
+    paramError("file", "both fixed radius and variable radius specified");
+  if (_fixed_radius && MooseUtils::absoluteFuzzyLessEqual(_radius, 0, libMesh::TOLERANCE * libMesh::TOLERANCE))
+    paramError("file", "fixed nucleus radius size must be greater than zero");
+
+    _console<<"found_r = "<<found_r<<", _fixed_radius = "<<_fixed_radius<<std::endl;
 
   // sort the nucleation history primarily according to time
-  std::sort(_nucleation_history.begin(), _nucleation_history.end());
+  //std::sort(_nucleation_history.begin(), _nucleation_history.end());
```

Comment:
please remove instead of commenting out

Reply:
Yes, I noted so far in my description to-do cleaning up comments, I will remove all these things.  But how do I handle the deprecation of the current system with the radius?